### ISSUE TITLE
The profiler doesn't show the dump result

### ISSUE THREAD

**Issue by glickel** [symfony/symfony#25283]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no
| Symfony version  | 4.0.0

In Symfony 4.0.0, with the WebProfilerBundle installed (composer require --dev profiler), when I do a dump($someVar) in a controller I can't see the result in the profiler.
Instead of that, the result is automatically put in the template.

Is it a real bug or the profiler will no longer show the dump result ?

**Comment by ro0NL** [comment]
Does `composer req debug` help? Im guessing the `symfony/debug-bundle` is not installed.

**Comment by glickel** [comment]
Yes that the solution, thank you :)

**Comment by nicolas-grekas** [comment]
Feel free to close :)

**Comment by aminvoly** [comment]
Hi.
I installed Symfony 4.1 and the dump function doesn't work on controllers but it works on twig files, if I remove the symfony/debug-bundle then the debug works properly in controller but it doesn't work in twig files. :D, do you guys have any idea why this is happening?

**Comment by xabbuh** [comment]
@aminvoly Can you open a new issue and also provide a small example application that allows to reproduce your issue?

**Comment by baris1892** [comment]
@aminvoly : I experienced the same issue on a fresh Symfony 4.1 installation (+ `symfony/debug-pack`), but it happens only if `APP_ENV=dev` in my `.env` file. For `APP_ENV=test` and `APP_ENV=prod` it works.

**Comment by baris1892** [comment]
@aminvoly : I got the same issue on a fresh symfony 4.1 installation (using symfony/debug-pack). However if I change the `.env` file to `APP_ENV=test` or `APP_ENV=prod` it works, but not for `APP_ENV=dev`. I've already cleared the caches, so that should be not the problem.

The following in a controller does not work (blank page is shown, nothing is dumped):
`      dump("dump from controller");
        dd("dd from controller");`
